paradime/cli/rich_text_output.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+
+from rich import box
+from rich.console import Console
+from rich.json import JSON
+from rich.panel import Panel
+from rich.table import Table
+from rich.text import Text
+
+console = Console()
+
+
+def print_cli_header(version: str) -> None:
+    version_title = Text(f"Paradime CLI ", style="green").append(f"v{version}", style="bold green")
+    version_title.append(
+        f"\n\nUse the Paradime CLI to login and trigger Bolt dbt schedules from the terminal.",
+        style="#9696a0",
+    )
+    version_title.append(f"\n\nRead examples of using the CLI and the SDK: ", style="#9696a0")
+    version_title.append(
+        f"https://github.com/paradime-io/paradime-python-sdk/tree/main/examples",
+        style="underline #9696a0",
+    )
+    console.print(Panel(version_title, padding=(1, 2), width=100), style="#827be6")
+
+
+def print_error_table(error: str, json: bool) -> None:
+    table = Table(border_style="#787885", box=box.SIMPLE, show_footer=True, width=100)
+    table.add_column("🚨 Error", justify="left", style="red", no_wrap=False)
+    if json:
+        error_text = JSON.from_data({"error": error}, highlight=False).text
+    else:
+        error_text = Text(error)
+    table.add_row(error_text, style="#f44336")
+    console.print(table)
```

Comment:
So the vision for the `json` boolean is that a 3rd party tool can consume it and easily extract the data from it using a json parser. If we just put it inside an ascii table it will be very hard for the users to pull it out - if that makes sense?

So I think we either just get rid of json if we don't want it or we do early escapes:
```suggestion
def print_error_table(error: str, json: bool) -> None:
    if json:
        click.echo({"error": error})
        return
    
    table = Table(border_style="#787885", box=box.SIMPLE, show_footer=True, width=100)
    table.add_column("🚨 Error", justify="left", style="red", no_wrap=False)
    error_text = Text(error)
    table.add_row(error_text, style="#f44336")
    console.print(table)
```

Same for the `print_run_started`  - lmk if you want me to do a PR into this quickly.

Reply:
gonna do a pr into here